misk-actions/api/misk-actions.api

```diff
@@ -193,7 +193,7 @@ public abstract interface class misk/web/ResponseBody {
 }
 
 public abstract interface annotation class misk/web/ResponseContentType : java/lang/annotation/Annotation {
-	public abstract fun value ()Ljava/lang/String;
+	public abstract fun value ()[Ljava/lang/String;
```

Comment:
This is binary-incompatible but I expect it’ll be fine 'cause you’ll very rarely have a different version of misk-actions to build against vs. run against.